nursery/reference-cryptocurrency-strings.yml

```diff
@@ -44,7 +43,3 @@ rule:
       - string: "Monero"
       - string: "Graft"
       - string: "Zcash"
-      - string: "Ya.money"
-      - string: "Ya.disc"
-      - string: "Steam"
-      - string: "vk.cc"
```

Comment:
don't appear to be related to cryptocurrency and I couldn't identify where they came from originally